cloud/prometheus/prometheus.conf

```diff
@@ -0,0 +1,33 @@
+# Global settings and defaults.
+global {
+  scrape_interval: "10s"
+  evaluation_interval: "10s"
+  labels {
+    label {
+      name: "service"
+      value: "super_duper"
```

Comment:
Well, it's the overall service [i.e. all the jobs get that label too], so and "...server" isn't quite right, but I've changed it to "ct" for now.